libs/deal-ticket/src/components/market-selector.tsx

```diff
@@ -249,8 +250,10 @@ export const MarketSelector = ({ market, setMarket, ItemRenderer }: Props) => {
 
   useEffect(() => {
     setResults(
-      data?.markets?.filter((item: MarketNames_markets) =>
-        item.name.match(new RegExp(escapeRegExp(lookup), 'i'))
+      data?.markets?.filter(
```

Comment:
i did this filtering once and i was wrong. in console v2 you can place limit orders on suspended (not active - continuously trading) markets that are in auction. there were times in testnet when all markets were suspended (in auction). 
if there are no active markets what is the plan to display in the market selector for console-lite?

Reply:
what we ended up doing was to filter the markets based on the MarketTradingMode not MarketState

Reply:
Currently we don't have such scenario possible in console lite app. We trade only active markets afaik. Component is displayed in view which is possible to reach for the user only if there is at least one active market. There wasn't such view in design, i could refer to. As component is placed in common library, I could invented something for such case, but from the other hand, I think it would be better to do it based on real scenario, if there were one in the future.

Reply:
https://github.com/vegaprotocol/frontend-monorepo/issues/656 check this out. I am working on order warning at the moment changing how `useOrderValidation` hook behaves. you guys are using it too.